### PR TITLE
NORALLY: graphman with no CLI-wrapper

### PR DESCRIPTION
**Pull request by graju256**
Running graphman with no additional CLI-wrapper
- only requirement is to install the module globally
- For more information, https://github.com/Layer7-Community/graphman-client?tab=readme-ov-file#getting-started-with-the-cli-
- publishing dev packages as SNAPSHOT artifacts

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -5,7 +5,8 @@ pipeline {
     environment {
         ARTIFACTORY_CREDS = credentials('ARTIFACTORY_USERNAME_TOKEN')
         ARTIFACTORY_ARTIFACT_PATH = 'usw1.packages.broadcom.com/artifactory'
-        ARTIFACTORY_ARTIFACT_NPM_PATH = "${env.ARTIFACTORY_ARTIFACT_PATH}/api/npm/apim-npm-dev-local/"
+        ARTIFACTORY_ARTIFACT_NPM_DEV_PATH = "${env.ARTIFACTORY_ARTIFACT_PATH}/api/npm/apim-npm-dev-local/"
+        ARTIFACTORY_ARTIFACT_NPM_RELEASE_PATH = "${env.ARTIFACTORY_ARTIFACT_PATH}/api/npm/apim-npm-dev-local/"
```
Comment (uppoju) [inline Jenkinsfile:9]: Dev and Release have same repos ?

Comment (graju256) [reply Jenkinsfile:9]: Let me correct this.

File: package.json [hunk 2]
```diff
@@ -1,14 +1,16 @@
 {
     "name": "@layer7/graphman",
-    "version": "1.3.1",
+    "version": "1.3.0",
```
[the remark this answers is not in the document]
Comment (graju256) [reply package.json:3]: I've just reset it to 0. Patch version comes from build number.

File: Jenkinsfile (hunk 1 above)
Comment (graju256) [reply Jenkinsfile]: @uppoju Done.

**Review by uppoju — APPROVED** [review]

**Pull request merged**